scripts/maintenance-status/cloud-create-mr

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3.6
```

Comment:
Why insist on Python 3.6? Does this not work with 3.7 or 3.8?

Reply:
it should, changed it to `python3` thanks :+1:

Reply:
Thanks!